src/CMakeLists.txt

```
@@ -2,63 +2,56 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ..)
-find_package(FlatBuffers REQUIRED)
-find_package(CLI11 REQUIRED)
-find_package(JSONForModernCPP REQUIRED)
-find_package(ConcurrentQueue REQUIRED)
-find_package(StreamingDataTypes COMPONENTS ce9a4745431e54bc45b0c927857a866fa77b84ec)
 find_package(CURL)
-find_package(fmt REQUIRED)
-find_package(GraylogLogger)
-find_package(StaticData COMPONENTS "schema-config-global.json")
+find_package(Threads)
 find_package(GitCommitExtract)
 
-set(path_include_common
-${CONCURRENTQUEUE_INCLUDE_DIR}
-${JSONFORMODERNCPP_INCLUDE_DIR}
-${CLI11_INCLUDE_DIR}
-${RDKAFKA_INCLUDE_DIR}
-${FLATBUFFERS_INCLUDE_DIR}
-${PROJECT_SOURCE_DIR}/src
-${PROJECT_BINARY_DIR}/src
-)
-
-set(path_include_common_suppressed_warnings
-        ${EPICSV4_INCLUDE_DIRS}
-        )
-
 set(libraries_common
-${path_library_epics_ca}
-${path_library_epics_pvData}
-${path_library_epics_pvAccess}
-${path_library_epics_NT}
-${RDKAFKA_LIBRARIES}
-${FETK_EXTRA_LIBRARIES}
-fmt::fmt
+        CONAN_PKG::librdkafka
+        CONAN_PKG::jsonformoderncpp
+        CONAN_PKG::concurrentqueue
+        CONAN_PKG::streaming-data-types
+        CONAN_PKG::cli11
+        CONAN_PKG::fmt
+        CONAN_PKG::flatbuffers
+        CONAN_PKG::asio
+        Threads::Threads
 )
 
-if (UNIX)
-  list(APPEND libraries_common pthread)
-endif()
+set(path_include_common "")
+
+# Allow a system installed EPICS library to be used instead of conan package (for ISIS-STFC)
+set(USE_EXTERNAL_EPICS FALSE CACHE BOOL "Look for system installed EPICS instead of using Conan package")
+if (USE_EXTERNAL_EPICS)
+    find_package(EPICSV4)
+    list(APPEND libraries_common
+            ${path_library_epics_ca}
+            ${path_library_epics_pvData}
+            ${path_library_epics_pvAccess}
+            ${path_library_epics_NT})
+    list(APPEND path_include_common ${EPICSV4_INCLUDE_DIRS})
+else()
+    list(APPEND libraries_common CONAN_PKG::epics)
+endif(USE_EXTERNAL_EPICS)
 
 set(compile_defs_common "")
 
 if (CURL_FOUND)
-	list(APPEND compile_defs_common "HAVE_CURL=1")
+    list(APPEND compile_defs_common "HAVE_CURL=1")
     list(APPEND libraries_common ${CURL_LIBRARIES})
     list(APPEND path_include_common ${CURL_INCLUDE_DIRS})
-endif()
+endif(CURL_FOUND)
 
-if (${GRAYLOGLOGGER_FOUND} AND ${USE_GRAYLOG_LOGGER})
+set(USE_GRAYLOG_LOGGER TRUE CACHE BOOL "Compile with support for logging to Graylog")
+if (CONAN_SPDLOG-GRAYLOG_ROOT AND USE_GRAYLOG_LOGGER)
     message(STATUS "Using graylog_logger")
     list(APPEND compile_defs_common "HAVE_GRAYLOG_LOGGER=1")
-    list(APPEND path_include_common ${GRAYLOGLOGGER_INCLUDE_DIR})
-    list(APPEND libraries_common ${GRAYLOGLOGGER_LIBRARY})
-elseif (NOT GRAYLOGLOGGER_FOUND AND USE_GRAYLOG_LOGGER)
-    message(WARNING "graylog_logger requested but not found")
-endif()
+    list(APPEND libraries_common
+            CONAN_PKG::spdlog-graylog)
+else()
+    list(APPEND libraries_common
+            CONAN_PKG::spdlog)
+endif(CONAN_SPDLOG-GRAYLOG_ROOT AND USE_GRAYLOG_LOGGER)
 
 set(INCLUDES
```

I very much prefer a (somewhat) clear indication of to which target a set of source files belong.
```suggestion
set(forwarder-library_INC
```